tests/space/test_space_objects.py

```diff
@@ -876,4 +876,4 @@ def test_get_space_tile_sampling(api):
     len_viz_low = len(tile_viz_low)
     len_viz_med = len(tile_viz_med)
     len_viz_high = len(tile_viz_high)
-    assert len_raw > len_viz_off > len_viz_low > len_viz_med >= len_viz_high
+    assert len_raw > len_viz_off >= len_viz_low > len_viz_med >= len_viz_high
```

Comment:
Same here, related to the topic of this PR?

Reply:
No, when I pushed the changes this test was failing hence, I fixed it along with my changes in a separate commit.